UKHomeOffice/sgmr-service
# SPCR-160 Remove unused babel plugins and update required ones

## Description
Review babel packages 
- updated used packages
- removed unneeded ones

## To test
- delete your `package-lock.json`
- delete your `node modules`
- run `npm i`
- start the app
- the app should run without errors


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
